ansible_collections/arista/avd/examples/single-dc-l3ls/README.md

```diff
@@ -256,86 +228,91 @@ all:
 
 ## Defining device types
 
-Since this example covers building a L3LS network, AVD must know about the device types, for example spines, L3 leaves, L2 leaves etc. The devices are already grouped in the inventory, so the device types are specified in the group variable files with the following names and content:
+Since this example covers building an L3LS network, AVD must know about the device types, for example, spines, L3 leaves, L2 leaves, etc. The devices are already grouped in the inventory, so the device types are specified in the group variable files with the following names and content:
 
-`ansible-avd-examples/single-dc-l3ls/group_vars/DC1_SPINES.yml`
+=== "DC1_SPINES.yml"
 
-```yml
----
-type: spine
-```
+    ```yaml
+    --8<--
+    examples/single-dc-l3ls/group_vars/DC1_SPINES.yml
+    --8<--
+    ```
 
-`ansible-avd-examples/single-dc-l3ls/group_vars/DC1_L3_LEAVES.yml`
+=== "DC1_L3_LEAVES.yml"
 
-```yml
----
-type: l3leaf
-```
+    ```yaml
+    --8<--
+    examples/single-dc-l3ls/group_vars/DC1_L3_LEAVES.yml
+    --8<--
+    ```
 
-`ansible-avd-examples/single-dc-l3ls/group_vars/DC1_L2_LEAVES.yml`
+=== "DC1_L2_LEAVES.yml"
 
-```yml
----
-type: l2leaf
-```
+    ```yaml
+    --8<--
+    examples/single-dc-l3ls/group_vars/DC1_L2_LEAVES.yml
+    --8<--
+    ```
 
-For example, this means that all switches that are children of the DC1_SPINES group defined in the inventory will be of the type `spine`.
+For example, all switches that are children of the DC1_SPINES group defined in the inventory will be of the type `spine`.
 
 ## Setting fabric-wide configuration parameters
 
-The `ansible-avd-examples/single-dc-l3ls/group_vars/FABRIC.yml` file defines generic settings that apply to all children of the `FABRIC` group as defined in the inventory described earlier.
+The `ansible-avd-examples/single-dc-l3ls/group_vars/FABRIC.yml` file defines generic settings that apply to all children of the `FABRIC` group as specified in the inventory described earlier.
 
 The first section defines how the Ansible host connects to the devices:
 
-```yml
-ansible_connection: ansible.netcommon.httpapi
-ansible_network_os: arista.eos.eos
-ansible_user: ansible
+```yaml title="FABRIC.yml"
+ansible_connection: ansible.netcommon.httpapi # (1)!
+ansible_network_os: arista.eos.eos # (2)!
+ansible_user: ansible # (3)!
 ansible_password: ansible
 ansible_become: true
-ansible_become_method: enable
-ansible_httpapi_use_ssl: true
-ansible_httpapi_validate_certs: false
+ansible_become_method: enable # (4)!
+ansible_httpapi_use_ssl: true # (5)!
+ansible_httpapi_validate_certs: false # (6)!
 ```
 
-- The Ansible host must use eAPI via HTTPS.
-- Network OS which in this case is Arista EOS.
-- The username/password combo.
-- How to escalate privileges to get write access.
-- Use SSL.
-- Do not validate SSL certificates.
+1. The Ansible host must use eAPI via HTTPS
```

Comment:
technically does not say use HTTPS - only HTTPAPI

Reply:
I'll update this to state "The Ansible host must use eAPI", thank you